packages/@lwc/engine/src/framework/base-lightning-element.ts

```diff
@@ -215,6 +217,7 @@ function BaseLightningElementConstructor(this: LightningElement) {
         vm.setHook = setHook;
         vm.getHook = getHook;
     }
+    defineProperty(component, membraneTag, EmptyObject);
```

Comment:
the tagging of the "blue" instance of the component is needed to support any value set in the constructors, and expandos in general. This is equivalent to what the observable membrane will do for any reactive proxy. The installed descriptor is non-configurable, non-writable, non-enumerable.